src/test/com/yetanalytics/flint/spec/axiom_test.cljc

```diff
@@ -8,77 +9,77 @@
+      (is (s/valid? ax/variable-spec '?0000))
+      (is (not (s/valid? ax/variable-spec '?foo.bar)))
+      (is (not (s/valid? ax/variable-spec '?.foobar)))
+      (is (not (s/valid? ax/variable-spec '?foo#bar)))
+      (is (not (s/valid? ax/variable-spec '?foo'bar)))
+      (is (not (s/valid? ax/variable-spec '?你好)))))
   (testing "blank nodes"
-    (is (ax/bnode? '_))
-    (is (ax/bnode? '_1))
-    (is (ax/bnode? '_foo))
-    (is (ax/bnode? '_foo.bar))
-    (is (not (ax/bnode? 'foo)))
-    (is (not (ax/bnode? '_.foobar)))
-    (is (not (ax/bnode? '_你好))))
+    (is (s/valid? ax/bnode-spec '_))
+    (is (s/valid? ax/bnode-spec '_1))
+    (is (s/valid? ax/bnode-spec '_foo))
+    (is (s/valid? ax/bnode-spec '_foo.bar))
+    (is (not (s/valid? ax/bnode-spec 'foo)))
+    (is (not (s/valid? ax/bnode-spec '_.foobar)))
+    (is (not (s/valid? ax/bnode-spec '_你好))))
   (testing "strings"
-    (is (ax/valid-string? "foo bar"))
+    (is (s/valid? ax/literal-spec "foo bar"))
     (testing "- double quotes"
-      (is (not (ax/valid-string? "\"")))
-      (is (ax/valid-string? "\\\"")))
+      (is (not (s/valid? ax/literal-spec "\"")))
+      (is (s/valid? ax/literal-spec "\\\"")))
     (testing "- back slashes"
-      (is (not (ax/valid-string? "\\")))
-      (is (ax/valid-string? "\\\\")))
+      (is (not (s/valid? ax/literal-spec "\\")))
+      (is (s/valid? ax/literal-spec "\\\\")))
     (testing "- line breaks"
-      (is (not (ax/valid-string? "\n")))
-      (is (not (ax/valid-string? "\\\n")))
-      (is (ax/valid-string? "\\n"))
-      (is (ax/valid-string? "\\\\n")))
+      (is (not (s/valid? ax/literal-spec "\n")))
+      (is (not (s/valid? ax/literal-spec "\\\n")))
+      (is (s/valid? ax/literal-spec "\\n"))
+      (is (s/valid? ax/literal-spec "\\\\n")))
     (testing "- carriage returns"
-      (is (not (ax/valid-string? "\r")))
-      (is (not (ax/valid-string? "\\\r")))
-      (is (ax/valid-string? "\\r"))
-      (is (ax/valid-string? "\\\\r")))
+      (is (not (s/valid? ax/literal-spec "\r")))
+      (is (not (s/valid? ax/literal-spec "\\\r")))
+      (is (s/valid? ax/literal-spec "\\r"))
+      (is (s/valid? ax/literal-spec "\\\\r")))
     (testing "- other escape input"
-      (is (ax/valid-string? "\\t"))
-      (is (ax/valid-string? "\\b"))
-      (is (ax/valid-string? "\\f"))
-      (is (not (ax/valid-string? "\\a")))
-      (is (not (ax/valid-string? "\\2")))))
+      (is (s/valid? ax/literal-spec "\\t"))
+      (is (s/valid? ax/literal-spec "\\b"))
+      (is (s/valid? ax/literal-spec "\\f"))
+      (is (not (s/valid? ax/literal-spec "\\a")))
+      (is (not (s/valid? ax/literal-spec "\\2")))))
   (testing "lang maps"
-    (is (ax/lang-map? {:en "foo"}))
-    (is (ax/lang-map? {:not-a-real-ltag "bar"}))
-    (is (not (ax/lang-map? {:en "\"foo\""})))))
+    (is (s/valid? ax/literal-spec {:en "foo"}))
+    (is (s/valid? ax/literal-spec {:not-a-real-ltag "bar"}))
+    (is (not (s/valid? ax/literal-spec {:en "\"foo\""})))))
```

Comment:
can you add a test for language maps with multiple kvs here and in other testing namespaces. I'm concerned that `com.yetanalytics.flint.axiom.impl.format/format-lang-map-literal` will drop data

I understand that multiple kvs corresponds to multiple triples 
e.g. `{:en "foo", :es "foo"} --> [[?s ?p "foo"@en] [?s ?p "foo"@es]]`
but I don't see how that conversion is handled

Reply:
If a map with multiple key-value pairs is used as a literal, it will just fail validation (as you can see [here](https://github.com/yetanalytics/flint/blob/protocols/src/main/com/yetanalytics/flint/axiom/impl/validation.cljc#L87)). Would be a good idea to add tests for that case, of course.

(I actually did think about how I would add support to multi-kv lang maps, since having to coerce them into sets of single-kv maps is rather annoying.)